bindings/nodejs/lib/types/block/output/irc-27.ts

```diff
@@ -0,0 +1,111 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { utf8ToHex } from '../../../utils';
+import { MetadataFeature } from './feature';
+
+/**
+ * The IRC27 NFT standard schema.
+ */
+class Irc27Metadata {
+    /** The IRC standard */
+    readonly standard: string = 'IRC27';
+    /** The current version. */
+    readonly version: string = 'v1.0';
+    /** The media type (MIME) of the asset.
+     *
+     * ## Examples
+     * - Image files: `image/jpeg`, `image/png`, `image/gif`, etc.
+     * - Video files: `video/x-msvideo` (avi), `video/mp4`, `video/mpeg`, etc.
+     * - Audio files: `audio/mpeg`, `audio/wav`, etc.
+     * - 3D Assets: `model/obj`, `model/u3d`, etc.
+     * - Documents: `application/pdf`, `text/plain`, etc.
+     */
+    type: string;
+    /** URL pointing to the NFT file location. */
+    uri: string;
+    /** The human-readable name of the native token. */
+    name: string;
```

Comment:
At least the 3 first fields should also be readonly I think

Reply:
I feel like there's no reason not to allow users to change them if they want, since this isn't a validated type